src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -49,56 +44,35 @@ Refer to the [four-step tutorial](https://axelar.network/blog/how-to-create-an-i
 
 To create a custom Interchain Token:
 
-1. Build your [ERC-20](https://docs.alchemy.com/docs/how-to-create-an-erc-20-token-4-steps) token and deploy it on multiple chains.
-2. Deploy a [Mint/Burn](/dev/glossary#mintburn) - [Token Manager](/dev/glossary#token-manager) for existing tokens on all chains using the [`deployTokenManager()`](https://github.com/axelarnetwork/interchain-token-service/blob/9edc4318ac1c17231e65886eea72c0f55469d7e5/contracts/InterchainTokenService.sol#L276) method on the Interchain Token Service specifying parameters like `salt`, `destinationChain`, [`tokenManagerType`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/ITokenManagerType.sol), `params`, and `gasValue`.
-3. Ensure that the Token Manager can invoke the `Mint/Burn` functions on the token by calling [`transferMintership()`](https://github.com/axelarnetwork/interchain-token-service/blob/9223108211b977d9138fdd67d5b4a641fc35c18c/contracts/utils/Minter.sol#L31) on your token. This will transfer the minter role to the Token Manager on all chains, enabling it to mint or burn tokens.
-   - Repeat this step on all other chains where the token exists, using the same deployer address and salt.
-4. Transfer Interchain Tokens between chains via the Interchain Token Service by calling [`interchainTransfer()`](https://github.com/axelarnetwork/interchain-token-service/blob/9223108211b977d9138fdd67d5b4a641fc35c18c/contracts/interfaces/IInterchainTokenService.sol#L210).
-
-```solidity
-/**
- * @notice Used to deploy TokenManagers.
- * @dev At least the `gasValue` amount of native token must be passed to the
- * function call. `gasValue` exists because this function can be part of a
- * multicall involving multiple functions that could make remote contract calls.
- * @param salt The salt to be used during deployment.
- * @param destinationChain The name of the chain to deploy the TokenManager and
- * standardized token to.
- * @param tokenManagerType The type of TokenManager to be deployed.
- * @param params The params that will be used to initialize the TokenManager.
- * @param gasValue The amount of native tokens to be used to pay for gas for the
- * remote deployment.
- * @return tokenId The tokenId corresponding to the deployed TokenManager.
- */
-function deployTokenManager(
-    bytes32 salt,
-    string calldata destinationChain,
-    TokenManagerType tokenManagerType,
-    bytes calldata params,
-    uint256 gasValue
-) external payable
-```
-
-To create the `params` that will be used to initialize the Token Manager in the `deployTokenManager` method above, you can use the [Online Ethereum ABI Encoder](https://adibas03.github.io/online-ethereum-abi-encoder-decoder/#/encode) specifying the operator of the Token Manager and the token address with the argument type `bytes` and `address` respectively.
```

Comment:
This was removed entirely and is not referenced anywhere. can we add it somewhere so developers knows what to do when creating `params`